zebra-network/src/meta_addr.rs

```diff
@@ -240,6 +240,20 @@ impl MetaAddr {
         self.last_seen
     }
 
+    /// Update reported last time for `NeverAttempted` peer by applying an offset.
+    ///
+    /// This adds the provided offset to the `last_seen` time reported by the peer that gossiped
+    /// this address. This is done to compensate for any clock differences between the local node
+    /// and the node that reported the address so that the time doesn't appear to be in the future.
+    ///
+    /// If the peer has already been interacted with by us, this method does nothing.
```

Comment:
Let's be really precise here - we don't have to have connected to the address, we just have to have tried.
```suggestion
    /// If the peer has been attempted, this method does nothing.
```